src/osm_configurator/view/toplevelframes/data_frame.py

```diff
@@ -19,19 +42,290 @@ class DataFrame(TopLevelFrame):
+                master=self,
+                text=str(self._selected_cut_out_path)
+            )
+        self._labels.append(self._cut_out_selected_path_label)
+
+        # Styles all labels the same way
+        for label in self._labels:
+            label.configure(
+                corner_radius=label_constants_i.LabelConstants.LABEL_CONSTANTS_CORNER_RADIUS.value,
+                fg_color=label_constants_i.LabelConstants.LABEL_CONSTANTS_FG_COLOR.value,
+                text_color=label_constants_i.LabelConstants.LABEL_CONSTANTS_TEXT_COLOR.value,
+                padx=label_constants_i.LabelConstants.LABEL_CONSTANTS_PADX.value,
+                pady=label_constants_i.LabelConstants.LABEL_CONSTANTS_PADY.value,
+                anchor=label_constants_i.LabelConstants.LABEL_CONSTANTS_ANCHOR.value,
+            )
+
+        # Implementing the checkbox
+
+        self._edge_building_are_in_checkbox: customtkinter.CTkCheckBox = \
+            customtkinter.CTkCheckBox(
+                master=self,
+                command=self.__edge_buildings_clicked,
+                fg_color=check_box_constants_i.CheckBoxConstants.CHECK_BOX_FG_COLOR.value,
+                hover_color=check_box_constants_i.CheckBoxConstants.CHECK_BOX_HOVER_COLOR.value,
+                text_color=check_box_constants_i.CheckBoxConstants.CHECK_BOX_TEXT_COLOR.value,
+                corner_radius=check_box_constants_i.CheckBoxConstants.CHECK_BOX_CORNER_RADIUS.value,
+                border_width=check_box_constants_i.CheckBoxConstants.CHECK_BOX_BORDER_WIDTH.value,
+                variable=tkinter.BooleanVar()
+            )
+
+        # Aligning everything nicely in the predefined grid
+
+        # Labels at the top
+        self._osm_data_select_label.grid(
+            row=1, column=1, rowspan=1, columnspan=1, padx=10, pady=10
+        )
+
+        self._cut_out_select_label.grid(
+            row=2, column=1, rowspan=2, columnspan=1, padx=10, pady=10
+        )
+
+        self._copy_label.grid(
+            row=4, column=1, rowspan=1, columnspan=1, padx=10, pady=10
+        )
+
+        # Buttons underneath the labels
+        self._osm_data_select_button.grid(
+            row=1, column=2, rowspan=1, columnspan=1, padx=10, pady=10
+        )
+
+        self._cut_out_select_button.grid(
+            row=2, column=2, rowspan=2, columnspan=1, padx=10, pady=10
+        )
+
+        self._copy_button.grid(
+            row=4, column=2, rowspan=1, columnspan=1, padx=10, pady=10
+        )
+
+        # Checkbox with label
+        self._edge_building_are_in_label.grid(
+            row=1, column=3, rowspan=2, columnspan=1, padx=10, pady=10
+        )
+
+        self._edge_building_are_in_checkbox.grid(
+            row=3, column=3, rowspan=1, columnspan=1, padx=10, pady=10
+        )
+
+        # Last button at the bottom
+        self._view_cutout_button.grid(
+            row=2, column=4, rowspan=2, columnspan=1, padx=10, pady=10
+        )
 
     def activate(self):
```

Comment:
you may have vorgotten to actually reloead the page, you are just getting the info needed, but dont set the info